benchmarks/hyperbahn-worker.js

```diff
@@ -107,10 +107,12 @@ HyperbahnWorker.prototype.createConfig = function createConfig() {
                 leafHost: '127.0.0.1',
                 leafPort: self.kafkaPort
             },
+            // 'clients.logtron.console': true,
```

Comment:
@Raynos this causes logtron to try to write into `/var/log/hyperbahn` which is an `EACCES` in travis.